install_script.sh.template

```diff
@@ -1122,6 +1127,10 @@ EOF
       formatted_host_tags="['""$( echo "$host_tags" | sed "s/,/','/g" )""']"  # format `env:prod,foo:bar` to yaml-compliant `['env:prod','foo:bar']`
       $sudo_cmd sh -c "sed -i \"s/^# tags:.*/tags: ""$formatted_host_tags""/\" $config_file"
   fi
+  if [ "$dd_env" ]; then
```

Comment:
```suggestion
  if [ -n "$dd_env" ]; then
```
we might want to replace the other ones (as per [recommendation](https://google.github.io/styleguide/shellguide.html#testing-strings), to prevent confusion use explicit tests)

Reply:
Agreed, I'll take a look to update similar checks